src/dialogs/image-cropper-dialog/image-cropper-dialog.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+  query,
+  unsafeCSS,
+  internalProperty,
+  PropertyValues,
+} from "lit-element";
+import "../../components/ha-dialog";
+import { haStyleDialog } from "../../resources/styles";
+import type { HomeAssistant } from "../../types";
+// @ts-ignore
+import cropperCss from "cropperjs/dist/cropper.css";
+import Cropper from "cropperjs";
+import { HaImageCropperDialogParams } from "./show-image-cropper-dialog";
+import "@material/mwc-button/mwc-button";
```

Comment:
😄

Reply:
teach those auto imports to add them at the right place 😄

Reply:
I wish that was a possible plugin/vscode feature